gusto_embedded/.speakeasy/speakeasy-modifications-overlay.yaml

```diff
@@ -0,0 +1,24 @@
+overlay: 1.0.0
+info:
+  title: Speakeasy Modifications
+  version: 0.0.2
+  x-speakeasy-metadata:
+    after: ""
+    before: ""
+    type: speakeasy-modifications
+actions:
+  - target: $["components"]["securitySchemes"]["SystemAccessAuth"]
+    update:
+      type: http
+      scheme: custom
+      x-speakeasy-custom-security-scheme:
```

Comment:
Overlays the SystemAccessAuth security scheme from our open api spec with a custom security scheme that takes a clientId and clientSecret as inputs. These are then used to fetch a system access token before each request, using the [clientcredentials hooks](https://github.com/Gusto/gusto-typescript-client/pull/7/files#diff-e009aa59985ff2f1d8b84e5762fd8be3f6ee369dccd237fb33db8184665a0743).